package.json

```diff
@@ -4,7 +4,7 @@
     "description": "A hapi plugin to aid migration for schwifty models",
     "main": "lib/index.js",
     "scripts": {
-        "test": "lab -a code -t 100 -L -I '__core-js_shared__,SharedArrayBuffer,Atomics'",
+        "test": "lab -a code -t 100 --timeout 10000 -L -I '__core-js_shared__,SharedArrayBuffer,Atomics'",
```

Comment:
What causes the need for the long timeout?

Reply:
For each test, we're connecting to the db and sometimes this can take some time (seems like the initial connection takes longer) -- to avoid false negatives in testing I upped the timeout

Reply:
10 seconds just seems really long for any test!  I guess we'll just have to keep an eye on it and see if there's any particular reason it might take that long.

Reply:
This was a shotgun approach -- I haven't ever seen it take 10 seconds -- just changed it to 3 seconds